### PR TITLE
lab1:completed

### PR DESCRIPTION
**Pull request by Srijanbista**
I have setup logic such that if the email field is empty, status field will have 'No Email' flag.

### CODE REVIEW

File: lab1.gs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+function myFunction() {
+  let excel = SpreadsheetApp.getActiveSpreadsheet();
+  let sheet = excel.getSheetByName("Sheet1");
+  let range = sheet.getRange("B2:E5");
```
Comment (ritushishir) [inline lab1.gs]: What if the sheet has entries beyond the 5th row? Please handle this case as well.

File: lab1.gs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+function myFunction() {
```
Comment (ritushishir) [inline lab1.gs]: You may want to rename this function!

**Review by ritushishir — CHANGES_REQUESTED** [review]

File: lab1.gs (hunk 2 above)
Comment (Srijanbista) [reply lab1.gs]: Sure!

File: lab1.gs (hunk 1 above)
Comment (Srijanbista) [reply lab1.gs]: Sure.

**Comment by ritushishir** [comment]
Srijan, please refer to this guidelines for folder structure in this repo
https://docs.google.com/presentation/d/1N9RpGUbO3qEDUenvfevhwaB38Roh-K6Gq80N3MwB0vI/edit#slide=id.g2538603901b_0_0

**Pull request closed without merging**